recipes/gstreamer/all/conanfile.py

```diff
@@ -0,0 +1,133 @@
+        del self.settings.compiler.cppstd
+
+    def config_options(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def build_requirements(self):
+        self.build_requires("meson/0.54.2")
+        self.build_requires("pkgconf/1.7.3")
+        if self.settings.os == 'Windows':
+            self.build_requires("winflexbison/2.5.22")
+        else:
+            self.build_requires("bison/3.5.3")
+            self.build_requires("flex/2.6.4")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("%s-%s" % (self.name, self.version), self._source_subfolder)
+
+    def _configure_meson(self):
+        meson = Meson(self)
+        defs = dict()
+        if self._is_msvc:
+            if tools.Version(self.settings.compiler.version) < "14":
+                defs["c_args"] = " -Dsnprintf=_snprintf"
+                defs["cpp_args"] = " -Dsnprintf=_snprintf"
+        if self.settings.get_safe("compiler.runtime"):
+            defs["b_vscrt"] = str(self.settings.compiler.runtime).lower()
+        defs["tools"] = "disabled"
+        defs["examples"] = "disabled"
+        defs["benchmarks"] = "disabled"
+        defs["tests"] = "disabled"
+        meson.configure(build_folder=self._build_subfolder,
+                        source_folder=self._source_subfolder,
+                        defs=defs, args=['--wrap-mode=nofallback'])
+        return meson
+
+    def build(self):
+        with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
+            meson = self._configure_meson()
+            meson.build()
+
+    def _fix_library_names(self, path):
+        # regression in 1.16
+        if self.settings.compiler == "Visual Studio":
+            with tools.chdir(path):
+                for filename_old in glob.glob("*.a"):
+                    filename_new = filename_old[3:-2] + ".lib"
+                    self.output.info("rename %s into %s" % (filename_old, filename_new))
+                    shutil.move(filename_old, filename_new)
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses", src=self._source_subfolder)
+        with tools.environment_append(VisualStudioBuildEnvironment(self).vars) if self._is_msvc else tools.no_op():
+            meson = self._configure_meson()
+            meson.install()
+
+        self._fix_library_names(os.path.join(self.package_folder, "lib"))
+        self._fix_library_names(os.path.join(self.package_folder, "lib", "gstreamer-1.0"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "lib", "gstreamer-1.0", "pkgconfig"))
+        tools.rmdir(os.path.join(self.package_folder, "share"))
+        tools.remove_files_by_mask(self.package_folder, "*.pdb")
+
+    def package_info(self):
+        self.cpp_info.includedirs = [os.path.join("include", "gstreamer-1.0")]
```

Comment:
I would just do an append.
This is the same behavior add when installing it in your Linux root.